Density/app/src/main/res/layout/facilities_activity.xml

```diff
@@ -99,6 +99,56 @@
                 android:layout_height="match_parent"
                 android:orientation="vertical">
 
+                <androidx.cardview.widget.CardView
+                    android:id="@+id/covidPolicyCard"
+                    style="@style/MyCardView"
+                    android:layout_width="fill_parent"
+                    android:layout_height="120dp"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginLeft="16dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="16dp"
+                    android:layout_marginRight="16dp"
+                    android:layout_marginBottom="8dp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/appbar">
+
+                    <ImageView
+                        android:id="@+id/masks_required_img"
+                        android:layout_width="120dp"
+                        android:layout_height="110dp"
+                        android:layout_marginLeft="40dp"
+                        android:layout_gravity="top|left"
+                        android:src="@drawable/masks_required"
```

Comment:
(I can't comment on the image file itself)

You should export and import the SVG version of the image from Figma to avoid blurriness on higher DPI phones.

To do that:

1. click on image in Figma
2. go to the "Export" section at the bottom of the right panel
3. click the "+", you should see a dropdown that allows you select "SVG"

![image](https://user-images.githubusercontent.com/11168401/94346956-3d184c00-fffe-11ea-8d6a-5960dae00640.png)